src/lib/theme/colors.ts

```diff
@@ -139,6 +139,34 @@ const palette: Readonly<Record<string, Record<Shade, HexColor>>> = {
     950: '#461B02',
     DEFAULT: '#F8A208',
   },
+  rhino: {
```

Comment:
Based on the naming convention in this file, I think we should name this something that better describes what color it is.

Reply:
Or should it just be `slate`?